pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -65,6 +65,9 @@ type criStatsProvider struct {
 	imageService internalapi.ImageManagerService
 	// hostStatsProvider is used to get the status of the host filesystem consumed by pods.
 	hostStatsProvider HostStatsProvider
+	//lint:ignore U1000 We can't import hcsshim due to Build constraints in hcsshim
+	// hcsshimInterface is used by Windows kubelet to gather stats for the for networking
+	hcsshimInterface interface{}
```

Comment:
I'm not too fond of using the hcsshim name explicitly here (or any of the other uses in the other files). Would windowsStatsInterface be a suitable replacement?

Reply:
Works for me, I don't really like the way this is interface is used but couldn't find another way to do this given the build constraints. Would be interested in feedback if it is worth this for having the unit tests.  The unit tests were helpful for me to get the logic right.

Reply:
updated

Reply:
@SergeyKanzhelev do you have any thoughts on if this is an ok approach to so unit tests can be enabled or should we drop this (and the unit tests).